Library/Homebrew/Gemfile

```diff
@@ -21,7 +21,7 @@ gem "tapioca"
 gem "activesupport"
 gem "concurrent-ruby"
 gem "mechanize"
-gem "patchelf"
+gem "patchelf", git: "https://github.com/rmnull/patchelf.rb", ref: "3a4fe5e8"
```

Comment:
This needs to be bumped to the latest work on https://github.com/david942j/patchelf.rb/pull/28, right?

Reply:
yep, working on it, ditching few commits, plus i'm not sure whether we wait for upstream to merge.
and then vendor it.
i think we better for @sjackman

Reply:
> yep, working on it, ditching few commits, plus i'm not sure whether we wait for upstream to merge.
> and then vendor it.

Let's keep this PR up-to-date with your branch. Once upstream merges it, we can vendor here.

Reply:
Agreed